config/locales/en.yml

```diff
@@ -503,6 +503,8 @@ en:
           attributes:
             form_type:
               blank: Select which form you need
+            language:
+              blank: Select which language you need the form in
```

Comment:
At the moment, the validation error, doesn't display `Select which language you need the form in`, when no radio buttons are selected. 

<img width="592" alt="Screenshot 2023-10-18 at 16 47 35" src="https://github.com/ministryofjustice/laa-estimate-financial-eligibility-for-legal-aid/assets/76905544/378e117a-580a-4d9c-afef-83e54cbe9694">



Also the validation error, doesn't disappear above the radio button when an option is selected. 

<img width="544" alt="Screenshot 2023-10-18 at 16 48 28" src="https://github.com/ministryofjustice/laa-estimate-financial-eligibility-for-legal-aid/assets/76905544/f0346505-67c4-4927-981a-c4719a9cfd83">

Reply:
My bad, a typo. Now fixed.